core/modules/date/tests/date_api.test

```diff
@@ -4,7 +4,7 @@
  * Test Date API functions
  */
 
-class DateAPITestCase extends BackdropUnitTestCase {
+class DateAPITestCase extends BackdropWebTestCase {
```

Comment:
This seems out of place. Is this correct?

Reply:
This was in the original PR from 2016, not sure why! I'll revert that change.